ArduCopter/defines.h

```diff
@@ -54,6 +54,7 @@ enum control_mode_t {
     FLOWHOLD  =    22,  // FLOWHOLD holds position with optical flow without rangefinder
     FOLLOW    =    23,  // follow attempts to follow another vehicle or ground station
     ZIGZAG    =    24,  // ZIGZAG mode is able to fly in a zigzag manner with predefined point A and point B
+    INITIALISING = 25,
```

Comment:
Comment required.  Don't know it helps all that much, but notionally we could generate the mavlink definitions from these (https://github.com/ArduPilot/mavlink/blob/master/message_definitions/v1.0/ardupilotmega.xml#L958)

Possibly using a script like this: https://github.com/peterbarker/ardupilot/commit/a495f91ae9eb3aca6bebb5b7cf5987df33f3ec13

Also - we'll want to add entries to the mavlink enumerations - but as you can see, we're already behind in that regard.

Reply:
Added comment in new commit